app/models/article.rb

```diff
@@ -75,6 +75,7 @@ class Article < ApplicationRecord
   before_save :fetch_video_duration
   before_save :clean_data
   before_save :update_cached_user
+  before_save :check_webmentions_support
```

Comment:
Since this is being run in a Sidekiq job and we expect the new `@canonical_url` to be present in that job this needs to be done in an `after_commit` callback to ensure the database is updated before the worker is kicked off.